inspirehep/modules/workflows/tasks/arxiv.py

```diff
@@ -63,14 +63,8 @@
 
 @with_debug_logging
 @backoff.on_exception(backoff.expo, DownloadError, base=4, max_tries=5)
```

Comment:
shouldn't the backoff be on the new download function (also?)?

Reply:
`download_document` never throws `DownloadError` explicitly. That's why I thought it was only necessary for this one.

Reply:
`download_file_to_workflow` which is used there, already retries for `ProtocolError`

Reply:
@michamos shall we :shipit: ?

Reply:
OK for me!